src/Geometry/AppendedTriangulations.jl

```diff
@@ -70,9 +70,9 @@ end
 #  lazy_append(a,b)
 #end
 
-function get_cell_shapefuns(trian::AppendedTriangulation)
```

Comment:
Here (in the `Geometry` mdoule) the name `get_cell_shapefuns` was already correct, since the result is the array of cell-wise shape functions.

Revert the renaming in this module and add `import Gridap.Geometry: get_cell_shapefuns` in `FESpaces.jl`

Reply:
ok, good call. Thanks for detecting this out!